### DISCUSSION TITLE
Tailwind Typography doesn't take in count Element Modifiers when a type scale is provided

### DISCUSSION THREAD

**Discussion by bachrc** [tailwindlabs/tailwindcss#9753] [Help] [answered]
Hello everyone,

I am using the Tailwind Typography plugin, and I want to customize the margins between each paragraph.

```
<article class="mt-12 mx-auto flex flex-col prose lg:prose-xl prose-p:mt-0 prose-p:mb-0">
```

Here, this will not work because of the lg:prose-xl. When I remove it, the margins are well removed. But not when it's here.. and I saw nothing on the docs or on the internet. Any ideas ?

**Comment by wongjn** [comment, marked as answer]
The behavior you are experiencing happens because the CSS rules generated by `lg:prose-xl` have the same specificity as the `prose-p:` rules, but the `lg:prose-xl` rules appear later in the source order and so if two rules have the same specificity, the one that appears later overrides any rules that appear before it. To counter this for your case, you can add `!important` to the `prose-p:` rules to ensure they are not overridden by the `lg:prose-xl` rules using the [important modifier](https://tailwindcss.com/docs/configuration#important-modifier), `prose-p:!mt-0 prose-p:!mb-0`. You can also write these classes more succinctly as `prose-p:!my-0`.

**Reply by bachrc** [reply]
Thanks a lot for your answer, I must admit I wouldn't have found this myself !
